qcodes/instrument/parameter.py

```diff
@@ -316,6 +322,38 @@ def __init__(self, name: str,
         # to perform a validation on get
         self._validate_on_get = False
 
+    @property
+    def raw_value(self) -> ParamRawDataType:
+        """
+        Represents the cached raw value of the parameter.
+
+        Setting the ``raw_value`` is not recommended as it may lead to
+        inconsistence state of the parameter.
+
+        Note that this property will be deprecated soon.
```

Comment:
You can have a look at step for how to get the getter and setter into the rendered docstring

Reply:
will do! thanks for noting!